nextflow.config

```diff
@@ -221,6 +221,17 @@ profiles {
         executor.cpus           = 4
         executor.memory         = 8.GB
     }
+    docker_self_hosted {
```

Comment:
Should we place it in `nf-test.config` so as to not cause confusion for the users?

Reply:
I am actually curious and think we don't need a special docker profile for GPU runners, because they are not running in rootless docker mode

Reply:
I am working on this PR. Will look into it as well. Thank you!

Reply:
Looks like we are not even using it. Will remove in the next commit!